tools/json_tools/gun_variant_validator.py

```diff
@@ -185,6 +185,7 @@
     "blunderbuss",
     "cannon",
     "carbine",
+    "civilian",
```

Comment:
These are all nouns for a firearm. Think I have a pistol/rifle/carbine - I have a civilian does not mean the same thing.
```suggestion
```